docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,297 @@
+
+When a NAP is `Accepted`, `Rejected`, or `Withdrawn`, the NAP should be
+updated accordingly. In addition to updating the status field, at the very
+least the `Resolution` header should be added with a link to the relevant
+post on the discussion forum.
+
+NAPs can also be `Superseded` by a different NAP, rendering the
+original obsolete. The `Replaced-By` and `Replaces` headers
+should be added to the original and new NAPs respectively.
+
+Process NAPs may also have a status of `Active` if they are never
+meant to be completed, e.g. NAP 0 (this NAP).
+
+### How a NAP becomes Accepted
+
+A NAP is `Accepted` by consensus of all interested contributors. We need a
+concrete way to tell whether consensus has been reached. When you think a
+NAP is ready to accept, start a topic on the Zulip naps channel with a
+subject like:
+
+> Proposal to accept NAP #\<number>: \<title>
+
+In the body of the topic, you should:
+
+- link to the latest version of the NAP,
+- briefly describe any major points of contention and how they were
+  resolved,
+- include a sentence like: "If there are no substantive objections
+  within 7 days from this post, then the NAP will be accepted; see
+  NAP 0 for more details."
+
+For an equivalent example in the NumPy library, see: <https://mail.python.org/pipermail/numpy-discussion/2018-June/078345.html>
+
+After you write the post, you should make sure to link to the specific
+thread from the `Discussion` section of the NAP, so that people can
+find it later.
+
+Generally the NAP author will be the one to make this post, but
+anyone can do it – the important thing is to make sure that everyone
+knows when a NAP is on the verge of acceptance, and give them a final
+chance to respond. If there's some special reason to extend this final
+comment period beyond 7 days, then that's fine, just say so in the
+post. You shouldn't do less than 7 days, because sometimes people are
+traveling or similar and need some time to respond.
+
+In general, the goal is to make sure that the community has consensus,
+not provide a rigid policy for people to try to game. When in doubt,
+err on the side of asking for more feedback and looking for
+opportunities to compromise.
+
+If the final comment period passes without any substantive objections,
+then the NAP can officially be marked `Accepted`. You should send a
+followup post notifying the thread (celebratory emoji optional but
+encouraged 🎉✨), and then update the NAP by setting its `:Status:`
+to `Accepted`, and its `:Resolution:` header to a link to your
+followup post.
+
+If there *are* substantive objections, then the NAP remains in
+`Draft` state, discussion continues as normal, and it can be
+proposed for acceptance again later once the objections are resolved.
+
+In unusual cases, when no consensus can be reached between core developers,
+the [napari Steering Council] may be asked to decide whether a
+controversial NAP is `Accepted`.
```

Comment:
Again I might link to the existing decision making process https://napari.org/community/governance.html#decision-making-process